auto_submit/test/requests/check_pull_request_test.dart

```diff
@@ -271,23 +347,27 @@ void main() {
     });
 
     test('Does not merge PR if no autosubmit label any more', () async {
```

Comment:
No need to push it back to pubsub as there is no `autosubmit` label any more.

Reply:
Yes Keyong, I modified the logic in check_pull_request handler to deal with this.